### PR TITLE
Support Webpack 2

### LINKED ISSUE #32: Webpack 2?

### PR DESCRIPTION
**Pull request by posva**
Closes #32

### CODE REVIEW

**Comment by posva** [comment]
I kept as many options as possible to be webpack 1 compatible. I also used Object.assign and removed the extend fn

File: index.js [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
+      highlight: function (str, lang) {
+        if (lang && hljs.getLanguage(lang)) {
+          try {
+            return hljs.highlight(lang, str).value
+          } catch (err) {}
+        }
+
         try {
-          return hljs.highlight(lang, str).value
+          return hljs.highlightAuto(str).value
```
Comment (unindented) [inline index.js:19]: Why not use the language that was given to us? It feels wasteful to do language detection again.

**Review by unindented — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (posva) [reply index.js:19]: I actually left that part of the code as it was 😄 
I think it's that is the fallback highlight

Comment (unindented) [reply index.js:19]: Ah sorry, now I see!

**Review by unindented — APPROVED** [review]

**Comment by unindented** [comment]
Published `0.6.0` with your changes. Thank you @posva !

**Comment by posva** [comment]
Thank you too! 😄

**Pull request merged**